### ISSUE TITLE
Groups based on 'Render Buffer' against entire layout

### ISSUE THREAD

**Issue by kevinsaucier** [xLightsSequencer/xLights#4256]
Some vendors (or maybe only Tom) utilize 'thirds' groups for sequencing.  Creating these 'thirds' groups is a pain, as it requires submodeling models to fit the thirds groups and, anytime a prop is moved around, it has to be redone.  Got me thinking that it would be handy if you could create a group that applied against coordinates on the main grid.  Basically, a group that has a start X/Y and an end X/Y (and maybe Z, if it seems worthwhile).  It would then be easy to create quarter/third/half groups, just like in the Render Buffer and they would only need to be modified if the width/height of the display change significantly.   Of course, just like with any other group, it would need to highlight the items in the group on the layout view so you could see what pixels were being 'captured' by the group.   Just a thought.

**Comment by derwin12** [comment]
Do you mean this?
![image](https://github.com/xLightsSequencer/xLights/assets/4643499/a708b55f-ceb6-4f14-808b-2759f5e5dc86)

**Comment by kevinsaucier** [comment]
Like that, but against the entire layout in one shot.    Here, you can see that I've moved stuff around so I need to repopulate my groups.  I've also had to split props in half for the group to work.  Would be nice to apply a 'buffer' that just encompassed whatever prop was moved into it.

![image](https://github.com/xLightsSequencer/xLights/assets/25715814/cc2469a3-ec65-408b-9895-9be58fcc8705)

**Comment by derwin12** [comment]
Wow ..okay,  that would be a big ask. Groups don't have anything like that - as you know, a group it simply brings together models or submodels etc.

**Comment by kevinsaucier** [comment]
Yup, but would pretty handy to be able to create groups like Garage, Front Lawn, Roof, etc., especially if you could create multiple X/Y points (like masking security cameras).   Just gotta pique the interest of the devs……  😁


Kevin
